build-pipelines-PR.yml

```diff
@@ -452,125 +160,54 @@ jobs:
       packDirectory: '$(Build.ArtifactStagingDirectory)/nuget'
       nobuild: true
       buildProperties: 'NuspecFile=bin\${{ parameters.BuildConfiguration }}\net6.0\win-x64\publish\dsc.nuspec'
-  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
-    displayName: 'Manifest Generator for nuget'
-    inputs:
-      BuildDropPath: '$(Build.ArtifactStagingDirectory)/nuget'
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: nuget'
-    inputs:
-      PathtoPublish: '$(Build.ArtifactStagingDirectory)/nuget'
-      ArtifactName: nuget
-  - task: CopyFiles@2
-    displayName: Collect Build Symbols - dsc
-    enabled: True
-    inputs:
-      SourceFolder: src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0
-      Contents: >-
-        */publish/dsc.?(pdb|exe|dll|)
-
-        */publish/Microsoft.BridgeToKubernetes.*.@(pdb|exe|dll|)
-
-        */publish/EndpointManager/EndpointManager.?(pdb|exe|dll|)
 
-        */publish/EndpointManager/Microsoft.BridgeToKubernetes.*.@(pdb|exe|dll|)
-      TargetFolder: $(Build.ArtifactStagingDirectory)/symbols
-      OverWrite: true
-  - task: PublishSymbols@2
-    displayName: Enable Source Server
-    condition: and(succeeded(), eq(variables.RunTests, 'true'))
-    enabled: True
-    inputs:
-      SymbolsFolder: $(Build.ArtifactStagingDirectory)/symbols
-      SearchPattern: '**/*.pdb'
-      SymbolServerType: TeamServices
-  - task: CopyFiles@2
-    displayName: Copy buildWindowsPDBs.ps1 to artifact directory
-    enabled: True
-    inputs:
-      SourceFolder: build
-      Contents: buildWindowsPDBs.ps1
-      TargetFolder: $(Build.ArtifactStagingDirectory)/symbols/
-      OverWrite: true
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: symbols'
-    condition: and(succeeded(), eq(variables.RunTests, 'true'))
-    enabled: True
-    inputs:
-      PathtoPublish: $(Build.ArtifactStagingDirectory)/symbols
-      ArtifactName: symbols
-      TargetPath: '\\my\share\$(Build.DefinitionName)\$(Build.BuildNumber)'
-  - task: tagBuildOrRelease@0
-    displayName: Tag BuildConfiguration
-    enabled: True
-    inputs:
-      tags: 'BuildConfiguration: ${{ parameters.BuildConfiguration }}'
 - job: Phase_2
-  displayName: Run NON self-contained build (binaries V2, where the binaries are downloaded in parallel)
+  displayName: Run PR build for non self contained
   timeoutInMinutes: 120
   cancelTimeoutInMinutes: 1
   pool:
     name: 'VSEngSS-MicroBuild2022-1ES'
     demands:
     - msbuild
     - visualstudio
-    - vstest
+    - DotNetFramework
```

Comment:
intentional? Curious why.

Reply:
I kept it similar to self-contained one's if you see above, it has all these three demands.

Reply:
ok we may want to update release pipeline too then?